src/app/(public)/robots/page.tsx

```diff
@@ -0,0 +1,30 @@
+import Navbar from "@/components/navigation/Navbar";
+import Footer from "@/components/sections/Footer";
+import Hero from "@/components/sections/home/Hero";
+import RobotPanel from "@/components/sections/robots/robot";
+import Image from "next/image";
+
+export const metadata = {
+	title: "Robots | FRC Team 2658",
+	description:
+		"Sponsorship information for FRC Team 2658, Σ-Motion, from Rancho Bernardo, California",
+};
+
+export default function Robot() {
+	return (
+		<main className="bg-black">
+			<div className="relative h-clampLarge overflow-hidden">
+				<Image
+					src="https://cdn.team2658.org/web-public/trident.jpeg"

```

Comment:
Find a different image to be used for the header. This is also used for the sponsors page. Here's the [link](https://photos.google.com/share/AF1QipNzr5b-u7YQkUZkXq43ViUqWDbSd5utQcOZF_AACi5gYFcWMKEyzsm5ji1aM8_Zag?pli=1&key=YlUwXzBrTkd5M3p5VGN6YmVoR3NEUWE4azFrOFh3) for 2024 Build Season photos, I can send ones from previous years later but we are having trouble accessing them right now.